Modules/Numerics/Optimizersv4/src/itkLBFGSOptimizerv4Base.cxx

```diff
@@ -214,7 +214,7 @@ LBFGSOptimizerBasev4<TInternalVnlOptimizerType>::GetStopConditionDescription() c
   }
   else
   {
-    return std::string("");
+    return { "" };
```

Comment:
Here, I would do either `return ""` or `return {}`.

Reply:
Done.  found a few other places in the code that similarly had return { "" } and replaced those too.